starlark/int.go

```diff
@@ -16,7 +16,7 @@ import (
 // Int is the type of a Starlark int.
 //
 // The zero value is not a legal value; use MakeInt(0).
-type Int struct{ impl intImpl }
+type Int struct{ intImpl }
```

Comment:
Thanks for pointing out the problem. Would you mind fixing it by instead making this change to int_generic.go:
```
-       return i.small_, i.big_
+       return i.impl.small_, i.impl.big_
```
I would prefer not to use embedding unnecessarily.

You'll need to sign Google's CLA before we can commit. Or if you like I can make the change directly.